helm/alfresco-content-services/charts/alfresco-sync-service/templates/config-syncservice.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     component: syncservice
 data:
+  {{ template "spring.activemq.config" . }}
```

Comment:
is syncservice a spring boot application? (I think not)

I don't think it requires both env vars and individual properties defined at L26

Reply:
No, it's not. I'm just reusing them to pass the value to the command line argument... Though I'm not sure why I didn't use the ACTIVEMQ_* ones instead.. 🤔. Will try that as it would be less confusing

Reply:
Ah I know remember why I did use the SPRING vars:
the activemq.config named template has a condition which rely on an activemq.enable value node. I kept this to avoid introducing more changes in the main chart but this node is not available to subcharts and I didn't want to have another template that would essentially do the same thing as spring.activemq.config just with different variable names.
I also don't want to spread activemq.enable to subcharts as 1) it doesn'treally make sense for them in their own context. é) I think we should skip a to model where feature flip relies on configuration being null or not (e.g. here messageBroker.url: null == activemq.enable: true).

Reply:
ok it's fine, just a little confusing to see the `spring.activemq.config` template used there.

Maybe an idea could be to define another template very similar to `spring.activemq.config` that provides env vars with the same values but with different name so it doesn't really look like we are using spring boot here

but since it's just for this maybe it's not worth the hassle

Reply:
I decided it was not worth it indeed

Reply:
> I didn't want to have another template that would essentially do the same thing as spring.activemq.config just with different variable names.

ok you already think about that xD